weather_mv/loader_pipeline/util.py

```diff
@@ -85,13 +85,15 @@ def to_json_serializable_type(value: t.Any) -> t.Any:
     # Note: The order of processing is significant.
     logger.debug('Serializing to JSON')
 
-    if pd.isna(value) or value is None:
+    # pd.isna() returns ndarray if input is not scalar therefore checking if not ndarray.
+    if type(value) != np.ndarray and (pd.isna(value) or value is None):
```

Comment:
What do you think about the following? 
```suggestion
    if (np.isscalar(value) and pd.isna(value)) or value is None:
```

If you prefer your method, my recommendation is to use `isinstance()` instead of type equality. I've been meaning to clean this up for a bit.

Reply:
I think using `np.isscalar` looks much cleaner and it handles other sequence objects like sets and tuples.

Reply:
Updated in https://github.com/google/weather-tools/pull/327/commits/dac23a047e2357b4f0dce9fb0b17969ae7f972a5.